lib/fuser_basic.h

```diff
@@ -31,19 +31,13 @@ namespace qsim {
  * User-defined controlled gates (controlled_by.size() > 0) and gates acting on
  * more than two qubits are not fused.
  * The template parameter Gate can be Gate type or a pointer to Gate type.
+ * This class is deprecated.
```

Comment:
What should previous users of this class use instead - I assume MultiQubitGateFuser? (Please add this in the comment here)

Reply:
Done.